src/spdx_tools/spdx/model/relationship.py

```diff
@@ -73,3 +73,6 @@ def __init__(
         comment: Optional[str] = None,
     ):
         check_types_and_set_values(self, locals())
+
+    def __hash__(self):
+        return hash("{} -> {} ({})".format(self.spdx_element_id, str(self.related_spdx_element_id), str(self.relationship_type)))
```

Comment:
I think our `Relationships` are mutable and that does not work well with hashing. This can cause inconsistencies when used in hashsets.

Reply:
This class needs a definition of __eq__ as well.